app/views/templates/payments/PaymentMessageHelper.scala

```diff
@@ -593,6 +593,34 @@ object PaymentMessageHelper {
     None
   )
 
+  object VATAAQuarterlyInstalLPI  extends PaymentMessageHelper(
+    VatAAQuarterlyInstalLPI.value,
+    "chargeType.VATAAQuarterlyInstalLPI",
+    Some("chargeType.for"),
+    Some("chargeType.for")
+  )
+
+  object VATAAMonthlyInstalLPI extends PaymentMessageHelper(
+    VatAAMonthlyInstalLPI.value,
+    "chargeType.VATAAMonthlyInstalLPI",
+    Some("chargeType.for"),
+    Some("chargeType.for")
+  )
+
+  object VATAAReturnCharge1stLPP extends PaymentMessageHelper(
+    VatAAReturnCharge1stLPP.value,
+    "chargeType.VATAAReturnCharge1stLPP",
+    Some("chargeType.for"),
+    Some("chargeType.for")
+  )
+
+  object VATAAReturnCharge2ndLPP extends PaymentMessageHelper(
+    VatAAReturnCharge2ndLPP.value,
+    "chargeType.VATAAReturnCharge2ndLPP",
+    Some("chargeType.for"),
+    Some("chargeType.for")
```

Comment:
I think these all need to be `chargeType.forPeriod`, so that they all read 'for period XXXX' in their descriptions